controller/cmd/service-mirror/cluster_watcher.go

```diff
@@ -271,7 +271,7 @@ func (rcsw *RemoteClusterServiceWatcher) cleanupOrphanedServices() error {
 
 	servicesOnLocalCluster, err := rcsw.localAPIClient.Svc().Lister().List(labels.Set(matchLabels).AsSelector())
 	if err != nil {
-		innerErr := fmt.Errorf("failed obtaining local services while GC-ing: %s", err)
+		innerErr := fmt.Errorf("failed obtaining source services while GC-ing: %s", err)
```

Comment:
this whole message could use some work, it's not clear what GC-ing is, exactly.

"failed to list services in source cluster while cleaning up mirror services:"